README.md

```diff
@@ -4,6 +4,13 @@ profile_service
 This app/solution can be run in 2 different environments
 
 ## Development
+### Graphics magick
```

Comment:
There is a library in NPM called Sharp that can be installed as a regular NPM package and can handle the image resizing.  It is also apparently more efficient then Graphics magick.
https://github.com/lovell/sharp